interface/tags_filters/start.php

```diff
@@ -86,4 +86,93 @@ function update_tags_filters()
 }
 add_action( 'update_plugin', 'update_tags_filters' );
 
-//}
```

Comment:
You need to un-comment line 5 and move this to the end of the file so it can at least be turned on and off with the global.